Opt-in updates (#3861)
MicrosoftDocs/azure-stack-docs: azure-local/update/update-opt-in.md

    | Hardware Vendor | SBE Family | Minimum Version Required | Required to be installed before or after 12.xx  |
    |--|--|--|--|
    | HPE | ProLiant-Minimal | 4.2.2507.1 | Either |
    | HPE | ProLiant-Standard | 4.2.2506.14 | Either |
    | Dell | AX-14G, AX-15G, or AX-16G | 4.2.2506.xx | Before |
    | Lenovo | ThinkAgileMXPremier or ThinkAgileMXStandard | 4.1.2505.10xx | Before (this will in turn require 4.2.2508.x SBE to be installed with 12.xx)  |
    | DataON | AZS1 or AZS6000 | 4.2.2504.x | Before (this will in turn require 4.2.2507.x SBE to be installed with 12.xx) |

## Opt-in update workflow

1. Connect to a machine on your Azure Local system using a user account that is part of the local administrator group, such as the deployment user.

1. Enable the solution package update flag by running the following command:

    ```powershell
    Enable-SolutionFeature -Name "24H2Ready"
    ```

1. Discover the update by running the following command a few times until you see `12.25xx.x.x` listed:

    ```powershell

>     Get-SolutionUpdate | Where-Object {$.State -like "Ready*" -or $.State -like "Additional*" -or $.State -like "HasPrereq*"} | FL DisplayName, Description, ResourceId, State, PackageType, Prerequisites

Reviewer: This is invalid syntax.  I'm getting on an env now and will provide valid syntax shortly.

Reviewer: This is valid syntax

```suggestion
    Get-SolutionUpdate | Where-Object {$_.State -like "Ready*" -or $_.State -like "Additional*" -or $_.State -like "HasPrereq*"} | FL DisplayName, Description, ResourceId, State, PackageType, Prerequisites
```

    ```

    > [!NOTE]

>     > If the 12.25xx update lists prerequisites, it typically indicates a specific SBE version (for example, 4.2.xx SBE) that must be installed first. This will appear in the output as `Ready` or `AdditionalContent` required. Follow the normal process to install the required SBE update, and then return to install the 12.25xx.x.x solution update.

Reviewer: "this will appear" is not clear what "this" is.

```suggestion
    > If the 12.25xx update lists prerequisites, it typically indicates a specific SBE version (for example, 4.2.xx SBE) that must be installed first. The prerequisite update will also appear in the list of update options with `Ready` or `AdditionalContent` required. Follow the normal process to install the required prerequisite update, and then return to install the 12.25xx.x.x solution update.
```

1. Perform the update using the [Azure Local update flow](./update-via-powershell-23h2.md) making sure to select the `12.25xx.x.x` package. Begin with [Step 2: Discover the updates](./update-via-powershell-23h2.md#step-2-discover-the-updates).

> 1. Once the update is finished, verify the OS version is updated to `26100.xxxx`.

Reviewer: I'm not sure where we expect customers to be viewing these 26100.xxxx type versions, but the below is straightforward and will report just 26100

```suggestion
1. Once the update is finished, verify the OS Build Number is updated to `26100` by using `Get-ComputerInfo | select OsBuildNumber`
```
